02/app.js

```diff
@@ -1,23 +1,31 @@
 const randomArray = createRandomArray();
 console.log(randomArray);
 
+for (let i = 0; i <= randomArray.length - 1; i++) {
```

Comment:
Chyba łatwiej jest też zapisać bez `=` oraz `-1` - mniej pisania, tj. `for (let i = 0; i < randomArray.length; i++) {`